### DISCUSSION TITLE
Add support for adding new saved replies

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#133087] [Mobile] [answered]
GitHub Mobile beta for Android

There's apparently a way to use a saved reply by scrolling the toolbar (below the editor area) to the left until ↰ is visible and then tapping it:

![Screenshot_20240722-081614.png](https://github.com/user-attachments/assets/203ddb56-819c-4c13-b800-ae6324c1a525)

That screen should include some interface to https://github.com/settings/replies

The simplest improvement would be just a simple link to open https://github.com/settings/replies in a browser. 

A fancier improvement would be too allow creating/editing saved replies in the app.

**Comment by iskae** [comment, marked as answer]
Hi @jsoref 👋 

You are right 👍 Created an internal tracking issue for supporting this on Mobile 🙇 

Cheers!
